tests/shared/pages/InboxPage.ts

```diff
@@ -40,21 +47,81 @@ export class ModalPage {
     await this.page.locator('.NotificationPwaModal__close-button').first().click()
   }
 
+  async getAddress() {
+    await this.page.locator('.Avatar').first().click()
+    await this.page.locator('wui-icon[name=copy]').first().click();
+    const copiedAddress: string = this.page.evaluate('navigator.clipboard.readText()') as any
```

Comment:
Clipboard doesn't work in some browsers. Also won't this race when running in parallel?

Would it be more reliable to get the address from the wallet UI instead?

Reply:
No longer using clipboard for this, instead now using alt text of avatar